test_runner/regress/test_remote_storage.py

```diff
@@ -446,12 +445,17 @@ def ensure_calls_started_grew():
             ), f"observations for {file_kind} {op_kind} did not grow monotonically: {observations}"
 
     def churn(data_pass1, data_pass2):
+        # overwrite the same data in place, vacuum inbetween, and
+        # and create image layers; then run a gc().
+        # this should
+        # - create new layers
+        # - delete some layers
         overwrite_data_and_wait_for_it_to_arrive_at_pageserver(data_pass1)
-        client.timeline_checkpoint(tenant_id, timeline_id)
-        client.timeline_compact(tenant_id, timeline_id)
         overwrite_data_and_wait_for_it_to_arrive_at_pageserver(data_pass2)
-        client.timeline_checkpoint(tenant_id, timeline_id)
-        client.timeline_compact(tenant_id, timeline_id)
```

Comment:
The pattern of double compaction repeats, but after the change to first do L0 => L1 and only then image creation... Well, I don't see why it would had ever been beneficial. The wait flush lsn preceds these anyways.